### DISCUSSION TITLE
Analytics 📈

### DISCUSSION THREAD

**Discussion by grantnorwood** [remix-austin/remixaustin-com#65] [Ideas]
We’d like to measure our engagement and success metrics, but what’s the best way to do that?

Let’s start a discussion!

## Requirements

1. Ability to measure the basic engagement metrics that Google Analytics captures _(page views duration, unique users, etc)_
2. Custom events _(for tracking button clicks and other interactions)_
3. Real-time dashboard
4. …

——

## Potential Solutions

* Google Analytics
* …

**Comment by sethdavis512** [comment]
Not really pushing the conversation forward that much buuuttt...here's the GA example from the official Remix examples repo:
https://github.com/remix-run/examples/tree/main/google-analytics

**Reply by brookslybrand** [reply]
Nice, yeah looks pretty easy.

I don't super have a dog in the fight, but I have been helping with a migration from GA to [Amplitude](https://amplitude.com/pricing) for work and been enjoying it a good bit, however it doesn't do quite as much auto-tracking as GA does I don't believe

**Reply by brookslybrand** [reply]
I might create a little POC and see what people think, just for funsies

**Reply by grantnorwood** [reply]
That would indeed be fun  :)  Maybe there's a really top-tier analytics provider who'll offer their product for free for our OSS project?

@brookslybrand If you'd like a `@remixaustin.com` email address, I can create one for you _(or just a forwarding email addy, I'm using that myself when writing companies to ask for free stuff)_.  Makes us seem more genuine with a domain name they can check into 🙂

**Reply by brookslybrand** [reply]
@grantnorwood sorry for the delay, would love a @remixaustin.com email! Definitely helpful for signing up for stuff

**Reply by grantnorwood** [reply]
Cool, I created `brooks at remixaustin dot com` for ya, it will forward to your Gmail.  You should get a verification email to enable it.

**Comment by brookslybrand** [comment]
Calling bankruptcy on all discussions and issues today. Going to start revamping the direction of this project, so stay tuned 🚀
